### PR TITLE
Bugfix/stack too deep

### PR DESCRIPTION
**Pull request by jomjamkusuma**

### CODE REVIEW

**Comment by jomjamkusuma** [comment]
average code coverage increased to 79% lines

File: contracts/NFTMarketplace.sol [hunk 1]
```diff
@@ -20,468 +20,466 @@ import "@openzeppelin/contracts/cryptography/MerkleProof.sol";
...
+  }
+
+  /// @notice create an order and deposit NFT to the contract
+  /// @param _orderIds ID for the order
+  /// @param _assetAddresses NFT contract address being listed
+  /// @param _tokenIds NFT token ID being listed
+  /// @param _is1155s NFT's being listed ERC1155 flag
+  /// @param _roots in the barter list in merkle tree root
+  // Stack too deep HERE!
+  //   function createBatchOrders(
```
Comment (pisuthd) [inline contracts/NFTMarketplace.sol:143]: Having an issue here doesn't we can remove the whole function.

Comment (pisuthd) [inline contracts/NFTMarketplace.sol:143]: Having an issue here doesn't we can remove the whole function.

Comment (jomjamkusuma) [reply contracts/NFTMarketplace.sol:143]: Okay, we can uncomment it; I commented on it for the test.

**Comment by pisuthd** [comment]
The source code are completely updated we may need it again later.

**Pull request closed without merging**